eoscompanion/routes.py

```diff
@@ -1136,19 +1207,45 @@ def _on_got_metadata_callback(load_metadata_error, metadata_bytes):
                                       'metadata',
                                       _on_got_metadata_callback)
 
+
+    def _on_got_application_info(_, result):
+        '''Callback function that gets called when we get the app info.'''
+        try:
+            app_info = EosCompanionAppService.finish_load_application_info(result)
+        except GLib.Error as error:
+            json_response(msg, {
+                'status': 'error',
+                'error': serialize_error_as_json_object(
+                    EosCompanionAppService.error_quark(),
+                    EosCompanionAppService.Error.INVALID_APP_ID,
+                    detail={
+                        'applicationId': query['applicationId'],
+                        'message': str(error)
+                    }
+                )
+            })
+            server.unpause_message(msg)
+            return
+
+        application_listing = application_listing_from_app_info(app_info)
+        content_db_conn.shards_for_application(application_listing,
+                                               callback=_on_got_shards_callback)
+
     logging.debug(
         'Get content metadata: clientId=%s, '
         'applicationId=%s, contentId=%s',
         query['deviceUUID'],
         query['applicationId'],
-        query['contentId'])
+        query['contentId']
+    )
```

Comment:
pylint did not like the indentation you're proposing here, and I had to change it to this style in order to fix the build the other day. See https://github.com/endlessm/eos-companion-app-integration/commit/8e8071fdf80d62e295a9abb2ddbddb4b9741ee2e

Reply:
Odd... I am pretty sure I had to do this to get pylint not to complain. I'll take another look.

Reply:
Yeah, pylint doesn't complain with this style, at least not from within my own flatpak builds?

Reply:
Ah, the pylint errors that you're referring to are because the later arguments had more indentation than the former ones (the end paren placement is actually flexible)